services/workflows-service/src/ui-definition/utils/collection-flow/8-website-basic-requirement.ts

```diff
@@ -31,9 +31,12 @@ const validationSchema = {
                       properties: {
                         mainWebsite: {
                           type: 'string',
-                          format: 'uri',
+                          pattern:
+                            '^((https?):\\/\\/)?([a-z0-9-]+\\.)+[a-z0-9]+(\\.[a-z]{2})?(\\/[a-zA-Z0-9#]+\\/?)?(\\?[a-zA-Z0-9_]+=[a-zA-Z0-9_]+(&[a-zA-Z0-9_]+=[a-zA-Z0-9_]+)*)?(#[a-zA-Z0-9_]+)?$',
```

Comment:
Found this regex helpul:
```^((http|https):\/\/.)[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&=]*)$```
and validate this regex here: https://regex101.com/

Reply:
@tomer-shvadron resolve and merge

Reply:
@liorzam @alonp99 How critical is it to change the regex right now? The regex I supplied has passed rigorous manual testing for all happy-flow scenarios I could think of and also a lot of bad-flow scenarios

Reply:
Also - Lior's regex is not got for our use-case as it enforces protocol. Haven't looked deeper to if "www." is optional or not, subdomains, querystring, query params, anchors... I think we should leave the regex as is.
Please tell me if you think otherwise